probe-rs/src/architecture/arm/core/armv7a.rs

```diff
@@ -948,6 +949,15 @@ impl CoreInterface for Armv7a<'_> {
         }
     }
 
+    fn endianness(&mut self) -> Result<Endian, Error> {
```

Comment:
I think if endianness is not expected to change in runtime, you can read this up once and cache it.

Reply:
PPC cores can have endianness that changes, but in this case I'll modify it so that it's read once and cached.